src/glass/src/app/pages/bootstrap-page/bootstrap-page.component.html

```diff
@@ -13,7 +13,7 @@
         <div fxLayout="column"
              fxLayoutAlign="center center">
           <p>
-            <span translate>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin porttitor quam convallis hendrerit lacinia. Integer consectetur metus eu nibh fringilla, ac lobortis nisl porttitor. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Donec at purus odio.</span>
+            <span translate>Aquarium will now install and setup the core system. All recognized storage devices are automatically assigned to the data pool. Afterwards you will be able to configure the services you want to provide.</span>
```

Comment:
I'd drop the second phrase. We don't have a single data pool, nor do we assign the devices to any pool whatsoever. While _I_ understand what you mean, I don't think it would be obvious to the user. :)